src/components/mainPages/Sidebar.vue

```diff
@@ -9,34 +9,31 @@
     >
       <ButtonIcon
         icon-name="add"
-        @click="_startCreatingItem"
+        @click="openItemForm"
       />
     </div>
-    <div
-      class="open-sidebar-button"
-      :class="{ hide: isSidebarOpen }"
-      :disabled="isSidebarOpen"
-      @click="_openSidebar(isLoggedIn ? 'lists' : 'sign up')"
-    >
-      <div class="arrow"></div>
-    </div>
-    <div
-      class="sidebar-mode-buttons"
-      :class="{ show: isSidebarOpen }"
-    >
-      <ButtonText
-        style-type="solid"
-        v-for="mode in sidebarModes"
-        :key="mode"
-        :active="sidebarMode === mode"
-        :text="mode"
-        @click="_openSidebar(mode)"
-      />
+    <div class="sidebar-buttons">
+      <div class="mode-buttons">
+        <ButtonText
+          style-type="solid"
+          v-for="mode in sidebarModes"
+          :key="mode"
+          :active="sidebarMode === mode"
+          :text="mode"
+          @click="_openSidebar(mode)"
+        />
+      </div>
+      <div class="open-button">
+        <ButtonSign
+          style-type="arrow"
+          @click="isSidebarOpen ? _closeSidebar() : _openSidebar(isLoggedIn ? 'lists' : 'sign up')"
```

Comment:
hmmm, strange. I remember that I commented it before. I think some mistake.
So.
I think that putting complicated conditions into template is not really good fore readability and about principle that for complicated logic we have methods.
Let's move it to a method.